pom.xml

```
@@ -1046,6 +1046,14 @@
 										</excludes>
 										<message>Prevent the use of both javax and jakarta packages</message>
 									</bannedDependencies>
+									<bannedDependencies>
+										<!-- the jms api now finally officially exists -->
+										<excludes>
+											<exclude>jakarta.jms:jakarta.jms-api:3.*</exclude>
```

Can you add an explanation why 3.* is excluded?

---

> Can you add an explanation why 3.* is excluded?

Omdat daarin de packages worden aangepast van javax.jms.* naar jakarta.jms.*

---

😉 ik bedoelde dus in de pom

---

Nou we willen dus niet dat die per ongeluk ergens wordt opgepakt als transitive dependency want dan krijgen we classes met "verkeerde" package namen.

---

That's for all deps that are version locked in the enforcer-plugin. I don't mind repeating it again but I'm not sure if it really adds anything here. Right now it's a fail fast mechanism, and prevents transient runtime dependencies.

---

Yes, I understand the mechanism. It's fine. I'll close it.